src/__tests__/components/MenuTabs.test.tsx

```diff
@@ -12,7 +13,11 @@ describe('MenuTabs', () => {
 
     const props: BottomTabBarProps = { navigation: fakeNavigation, state: { index: 0, routes: [] } } as any
 
-    const actual = renderer.render(<MenuTabs {...props} />)
+    const actual = renderer.render(
+      <SafeAreaProvider>
```

Comment:
Once we merge #4033, you can simply revert the entire `__tests__` folder, since all the deep tests will now use `FakeProvider`. Then `FakeProvider` can be edited as follows:

```ts
return (
    <SafeAreaProvider initialMetrics={initialMetrics}>
      <Provider store={store}>{children}</Provider>
    </SafeAreaProvider>
  )
}

// These match what our old react-native-safe-area-view library returned:
const initialMetrics: Metrics = {
  frame: { height: 1334, width: 750, x: 0, y: 0 },
  insets: { bottom: 0, left: 0, right: 0, top: 20 }
}
```

With this in place, even the snapshots remain unchanged.